stream_alert_cli/helpers.py

```diff
@@ -110,12 +110,14 @@ def tf_runner(**kwargs):
     kwargs:
         targets: a list of Terraform targets
         action: 'apply' or 'destroy'
+        refresh_state: True or False
 
     Returns:
         bool: True if the terraform command was successful
     """
     targets = kwargs.get('targets', [])
     action = kwargs.get('action', None)
+    refresh_state = kwargs.get('refresh_state', False)
```

Comment:
This should default to `True`, since we only want to disable refreshing during latter deploy steps

Reply:
Fixed.